### DISCUSSION TITLE
Are infinite aliases allowed?

### DISCUSSION THREAD

**Discussion by roke-julian-lockhart** [anonaddy/anonaddy#461] [Q&A] [answered]
I currently use one e-mail address and sub-address to each service I use it for. I have, however, 2000 accounts, so even a perfectly sub-addressed e-mail address routed via anonaddy.me isn't enough.

Looking at https://github.com/anonaddy/anonaddy/discussions/168, I realized that to implement that feature would require a user to be able to create as many e-mail addresses per account as they desire.

However, currently, this doesn't appear to be possible.

**Comment by roke-julian-lockhart** [comment]
I just checked https://app.anonaddy.com/settings/subscription again:

> Unlimited Standard Aliases

is for all tiers.

Can someone confirm that this is now possible? I swear it wasn't previously!

**Reply by willbrowningme** [reply, marked as answer]
It has always been possible to create unlimited aliases using your username subdomain *@username.anonaddy.com.

The Pro plan can also create unlimited shared domain aliases as seen on the [pricing section](https://anonaddy.com/#pricing) of the home page.

**Reply by roke-julian-lockhart** [reply]
Thanks, @willbrowningme. That's... incredible. I can finally organize everything.
